Dockerfile

```diff
@@ -1,20 +1,21 @@
-FROM golang:1.15 as builder
+FROM golang:1.16 as builder
```

Comment:
This is necessary because the `go-dockerclient` has switched to `1.16`. Without this change the dependency wouldn't build.